internal/pkg/rpaas/manager.go

```diff
@@ -252,6 +262,43 @@ type CertificateData struct {
 	Key         string `json:"key"`
 }
 
+func getAnnotations(params map[string]interface{}) (annotations map[string]string) {
```

Comment:
There are a bunch of validations here. It'd better returning an error in this function in order to make the client notice what happened.

Reply:
You're right. At first glance seems to be a first idea just try to ignore them but it's better return it to client